frontend/public/module/k8s/swagger.ts

```diff
@@ -31,32 +30,14 @@ export const getStoredSwagger = (): SwaggerDefinitions => {
   }
 };
 
-const isSwaggerStale = () => {
-  // Avoid refreshing the entire document on cellular connections.
-  const connection = (navigator as any).connection || (navigator as any).mozConnection || (navigator as any).webkitConnection;
-  if (connection && connection.effectiveType === 'cellular') {
-    return false;
-  }
-  // Store the swagger definitions in localStorage, but refresh once per day.
-  // This document is large, so avoid reloading it too often.
-  const lastUpdated = Number(window.localStorage.getItem(SWAGGER_TIMESTAMP_LOCAL_STORAGE_KEY));
-  return !_.isFinite(lastUpdated) || lastUpdated < (Date.now() - (1000 * 60 * 60 * 24));
-};
-
 const storeSwagger = (swagger: SwaggerAPISpec) => {
   // Only store definitions to reduce the document size.
   const json = JSON.stringify(swagger.definitions || {});
```

Comment:
Just curious, is there a way that we can use Etags to conditionally update the stored JSON (and is it worth doing)? Otherwise, aren't we stringifying this on every request, regardless of whether it's cached or not?